webdistill.css

```
@@ -19,7 +55,6 @@ border: 2pt solid blue;
 
 .search {
 	position: relative;
-	max-width: 400px;
 	padding-top: 10pt;
 	padding-left: 10pt;
 	padding-bottom: 10pt;

```

You can use [shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties) to turn this into one line. 
```suggestion
	padding: 10px 0 10px 10px;
```